kaska/s1_observations.py

```diff
@@ -140,29 +148,29 @@ def read_time_series(self, time_grid):
         late = time_grid[-1]
 
         sel_dates = [k for k, v in self.dates.items()
-                     if (early <= k <= late)]
+                     if early <= k <= late]
         sel_bands = [v for k, v in self.dates.items()
-                     if (early <= k <= late)]
+                     if early <= k <= late]
         obs = {}
-        for ii, layer in enumerate(self.s1_data_ptr.keys()):
+        for my_ii, layer in enumerate(self.s1_data_ptr.keys()):
```

Comment:
Perhaps `j` as a loop dummy variable, since there is an `i` in an inner context.

Reply:
Changed